django-ydb-backend/pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[project]
+name = "django-ydb-backend"
+version = "0.1.0"
+description = ""
```

Comment:
let's add a small description what we are going to implement here (YDB Django Backend)